src/Catalog/NuGet.Services.Metadata.Catalog.csproj

```diff
@@ -268,6 +270,9 @@
     <SignPath>..\..\build</SignPath>
     <SignPath Condition="'$(BUILD_SOURCESDIRECTORY)' != ''">$(BUILD_SOURCESDIRECTORY)\build</SignPath>
     <SignPath Condition="'$(NuGetBuildPath)' != ''">$(NuGetBuildPath)</SignPath>
+    <SignType Condition="'$(SignType)' == ''">none</SignType>
```

Comment:
Added this assignment to guarantee that we never get in a state where signing happens despite `SignType` being unspecified.